rest_blog/serializers.py

```diff
@@ -0,0 +1,20 @@
+from django.utils import timezone
+from rest_framework import serializers
+from blog.models import Reminder
+
+
+class ReminderSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Reminder
+        fields = ('id', 'title', 'message', 'reminder_time', 'created_time', 'modified_time')
+
+    def validate_reminder_time(self, reminder_time):
```

Comment:
@vamsikrishna-kolli Good use of validate method. Always remember not to commit print statements.